compiler/rustc_target/src/abi/call/mod.rs

```diff
@@ -422,6 +426,11 @@ impl<'a, Ty> TyAndLayout<'a, Ty> {
                 }))
             }
 
+            Abi::ScalableVector { .. } => Ok(HomogeneousAggregate::Homogeneous(Reg {
+                kind: RegKind::ScalableVector,
+                size: Size::from_bits(128),
```

Comment:
This size is a lie. Would panicking or returning `Err` instead work? Or is it genuinely possible for a struct that is passed as argument to contain a scalable vector?

Reply:
In the RFC I did state that these types can't be inside a struct (currently no checking to enforce that though). We might want to relax that in the future though. Currently we could panic or return an error here, I'll make this an error for now.